bundle/Controller/DownloadController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
I think we cannot use PHP 7 features, since this bundle still supports PHP 5.

@MarioBlazek ?

Reply:
@darinda @emodric yes, we still need to support PHP 5.